style.css

```diff
@@ -0,0 +1,390 @@
+@font-face {
+  font-family: NexaLight;
+  src: url('./Font/NexaLight.otf');
+}
+
+@font-face {
+  font-family: NexaBlack;
+  src: url('./Font/NexaBlack.otf');
+}
+
+@font-face {
+  font-family: NexaBold;
+  src: url('./Font/NexaBold.otf');
+}
+
+* {
+  margin-top: 10px;
+}
+
+.banner-section {
+  background-color: black;
```

Comment:
when you give color in bg than what is the need of giving here @aeshapatel025

Reply:
changes done